jwst/master_background/master_background_step.py

```diff
@@ -73,19 +78,23 @@ def process(self, input):
 
             # Check if user has supplied a master background spectrum.
             if self.user_background is None:
-                pass
+                # Return input as dummy result for now
+                result = input_data
             else:
-                pass
+                # Return input as dummy result for now
+                result = input_data
+
+                # Record name of user-supplied master background spectrum
+                try:
+                    result.meta.background.master_background = basename(self.user_background)
```

Comment:
Doh!  My test didn't catch this.  Weird.

Reply:
The test didn't catch this because `ModelContainer` contains `meta`, and will happily accept whatever you put into it, instead of rejecting it in the `try/except` block and putting the result in each model of the `ModelContainer`.